### PR TITLE
WIP: Setting PKG_CONFIG_PATH for Qt configure command

### PR DESCRIPTION
**Pull request by chetnieter**
Adding PKG_CONFIG_PATH definition to the Qt configure command.
This forces the Qt configure script to look in the fletch install
directory for libraries.

### CODE REVIEW

**Comment by chetnieter** [comment]
@mwoehlke-kitware @dstoup This simple change fixed my issue with Qt finding an older system libpng over the fletch installed.

File: CMake/External_Qt.cmake [hunk 1]
```diff
@@ -132,7 +132,7 @@ if(WIN32)
     endif()
   endif()
 else()
-  set(Qt_configure ./configure)
+  set(Qt_configure PKG_CONFIG_PATH=${fletch_BUILD_INSTALL_PREFIX}/lib/pkgconfig ./configure)
```
Comment (mwoehlke-kitware) [inline CMake/External_Qt.cmake]: I wouldn't disbelieve this works for you, and it *might* always work, but we might want to do more like we do for the build command, i.e. hoist `set(env ${CMAKE_COMMAND} -E env)` to earlier and have a `build_env_var` and `configure_env_var`? Also, we should probably preserve an existing `PKG_CONFIG_PATH`...

**Comment by chetnieter** [comment]
@mwoehlke-kitware Changed things to better mirror what was done for the Qt build command. Let me know if this addresses your concerns.

**Comment by dstoup** [comment]
I haven't verified anything. Was waiting until the MR settled before doing that. That said, I never experienced the problem so my verification will be something like 'it didn't break anything'

**Comment by dstoup** [comment]
FWIW, soon we will have a dashboard that enables Qt5 as default. That will help us 'verify' every fix like this, once it's up.

**Comment by mwoehlke-kitware** [comment]
>> assuming you've verified it works after the changes
>
> I haven't verified anything.

That was directed at Chet :slightly_smiling_face:. Reason for @mentioning you was to check that you're also okay with the changes, i.e. "@dstoup, please review this also".

**Comment by dstoup** [comment]
Changes look fine to me.

**Comment by chetnieter** [comment]
Yes - I checked and with the new changes Qt picks up the fletch installed libpng on my linux box and the my centos 7 docker image I am working on.

**Pull request merged**